x/bank/module.go

```diff
@@ -106,6 +106,10 @@ func (am AppModule) RegisterServices(cfg module.Configurator) {
 	if err := cfg.RegisterMigration(types.ModuleName, 2, m.Migrate2to3); err != nil {
 		panic(fmt.Sprintf("failed to migrate x/bank from version 2 to 3: %v", err))
 	}
+
+	if err := cfg.RegisterMigration(types.ModuleName, 3, m.Migrate3to4); err != nil {
+		panic(fmt.Sprintf("failed to migrate x/bank from version 3 to 4: %v", err))
+	}
```

Comment:
I'm pretty sure that, for our fork, this needs to be migration 2 and the one above that's 2, should be the new 3.

Need to double check what it looks like in v0.45 of our fork to be sure.